Gemfile

```diff
@@ -23,7 +23,7 @@ gem "public_activity", "~> 1.5"
 gem "pundit"
 gem "rollbar"
 gem "rails", "~> 6.0.3"
-gem "redis", "~> 4.2"
+gem "redis", "< 4.2"
```

Comment:
The commit message says "the Redis version, which was pinned to `> 4.2` in Sidekiq 5.2.9". It should say "< 4.2", right?

Reply:
Whoops! Fixed